src/applications/gi/components/profile/CalculateYourBenefitsForm.jsx

```diff
@@ -97,6 +98,8 @@ function CalculateYourBenefitsForm({
       }
       setInvalidZip('');
       setInputUpdated(true);
+
+      if(!environment.isProduction()) updateEstimatedBenefits();
```

Comment:
looks like a few linting errors to sort out.

Reply:
Fixed